src/Sylius/Bundle/CoreBundle/Controller/CheckoutController.php

```diff
@@ -194,11 +194,12 @@ public function finalizeAction(Request $request, OrderInterface $order)
     }
 
     /**
+     * @param int $id
+     *
      * @return Response
      */
-    public function thankYouAction()
+    public function thankYouAction($id)
```

Comment:
We should not be able to see every order thank you page... This should be either in sesssion (for guests) or search by id and currently logged in customer?